packages/nextjs/src/client/routing/appRouterRoutingInstrumentation.ts

```diff
@@ -68,20 +60,13 @@ export function appRouterInstrumentation(
 
       prevLocationName = transactionName;
 
-      if (activeTransaction) {
-        activeTransaction.end();
-      }
-
-      const transactionContext = {
+      startNavigationSpanCallback({
         name: transactionName,
         op: 'navigation',
```

Comment:
l: We can do this later, but we should rewrite this to use all-attributes here instead of tags, metadata, op, origin.

Reply:
(same above)